polyphemus/lib/polyphemus.rb

```diff
@@ -17,4 +17,31 @@ def setup_db
     @db.extension :pg_json
     @db.pool.connection_validation_timeout = -1
   end
+
+  def setup_yabeda
+    Yabeda.configure do
+      group :polyphemus do
+        gauge :last_command_completion do
+          comment "Unix time of last time command was completed"
+          tags [:command, :status]
+        end
+      end
+    end
+
+    super
+  end
+
+  def run_command(config, *args, &block)
+    cmd, cmd_args, cmd_kwds = find_command(*args)
+
+    begin
```

Comment:
Here is one example where I've integrated metrics into the polyphemus command to watch for failed commands (right now that includes the mvir1 job).  This can be improved and wrapped up better over time, this is an example pass.